predict.py

```diff
@@ -83,17 +83,7 @@ def handler(request, context):
     pil_img = Image.open(f)
     pil_img = np.asarray(pil_img).astype(np.float32)
     height, width, _channels = pil_img.shape
-
-    # TODO: currently image smaller than IMG_SIZE will not be handled properly.
-    if height < Parameters.IMG_SIZE or width < Parameters.IMG_SIZE:
```

Comment:
removed because any image size can be handled

Reply:
> pil_img = np.asarray(pil_img).astype(np.float32)

たまにRGBじゃないデータがあったりするので、よく
`pil_img = np.asarray(pil_img.convert('RGB')).astype(np.float32)`
って書いてます（コメント)

Reply:
> pil_img = np.asarray(pil_img.convert('RGB')).astype(np.float32)

Good practice! Thank you!

Reply:
> removed because any image size can be handled

👍 
BTW, Did you resize the image to the appropriate size (Parameters.IMG_SIZE x Parameters.IMG_SIZE) when you did a prediction? or You can resize the image here.